config/m3aggregator.yaml

```diff
@@ -61,6 +61,7 @@ http:
   writeTimeout: 45s
 
 aggregator:
+  timerAggregationTypes: Sum,SumSq,Mean,Lower,Upper,Count,Stdev,Median,P50,P95,P99
```

Comment:
Add a comment here saying they need to be comma deliminated string? Might be more consistent to make it a list like the quantiles, though